cs/v1/CS_v1.0.0.yaml

```diff
@@ -514,12 +512,133 @@ paths:
     parameters: []
 components:
   schemas:
-    Place:
-      title: Place
+    PlaceOfReceipt:
+      title: Place of Receipt
       type: object
-      description: The Location specifying where the place is located.
-      x-stoplight:
-        id: 6qncmwke0iqvq
+      description: The Location specifying where the place of receipt is located.
+      required:
+        - facilityTypeCode
+        - location
+        - dateTime
+      properties:
+        facilityTypeCode:
+          description: |
+            The code to identify the specific type of facility. The code indicates which role the facility plays during the transportCall.
+            - `BORD` (Border)
+            - `CLOC` (Customer Location)
+            - `COFS` (Container Freight Station)
+            - `OFFD` (Off Dock Storage)
+            - `DEPO` (Depot)
+            - `INTE` (Inland Terminal)
+            - `POTE` (Port Terminal)
+            - `PBPL` (Pilot Boarding Place)
+            - `BRTH` (Berth)
+            - `RAMP`  (Ramp)
+            - `WAYP` (Waypoint)
+          example: POTE
+          type: string
```

Comment:
Since we changed these properties to not be Enum any more - I think you should add a `maxLength: 4`